flow/Platform.actor.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Platform.actor.h
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2020 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+// When actually compiled (NO_INTELLISENSE), include the generated version of this file.  In intellisense use the source version.
+#if defined(NO_INTELLISENSE) && !defined(FLOW_PLATFORM_ACTOR_G_H)
+        #define FLOW_PLATFORM_ACTOR_G_H
+        #include "flow/Platform.actor.g.h"
+#elif !defined(FLOW_PLATFORM_ACTOR_H)
+        #define FLOW_PLATFORM_ACTOR_H
+
+#include <stdlib.h>
+#include <string>
+#include <vector>
+
+#include "flow/flow.h"
+#include "flow/Platform.h"
```

Comment:
What is the new model for including Platform.h supposed to look like? Ideally, I think we'd only need to include one or the other everywhere. Since Platform.h is the one we had been including, is it feasible to make that behavior the same (i.e. you get Platform.actor.h by including Platform.h)? I see we are including Platform.h here, though I'm not sure if that's a requirement.

Reply:
I attempted this, and discussed this idea with Markus.  We eventually will want just one .h file, but we decided trying to consolidate these files is a complicated project that will require some work and discussion, and therefore is out of scope of this PR

Reply:
I agree that consolidating them would be nice and am ok with deferring that if it's complicated, but the question here is hopefully more straightforward. Specifically, could it be the case that `Platform.h` includes `Platform.actor.h` such that any users of platform would always just include `Platform.h`?

Reply:
I don't think this will work (or at least is not straight-forward). One difficulty is that `Platform.h` is included in several C files and actors don't compile in that context. One quick workaround would be (if I don't miss something, which I might):

```
#ifdef __cplusplus
#include "flow/Platform.actor.h"
#endif
```

Reply:
I see, I guess in that case we can leave this be for now.